032-MLOpsFromScratch/Coach/00-prereqs.md

```diff
@@ -0,0 +1,36 @@
+# Challenge 0: Pre-requisites
+
+**[Home](../README.md)** - [Next Challenge >](./01-TimeSeriesForecasting.md)
+
+## Solution 
+
+In this challenge we'll be setting up all the tools we will need to complete our challenges, preferably done a few days before the event. Additionally, we have a [presentation](MLOps%20Lectures.pptx) that can be used to help give a high level introduction to MLOps for the students, at both a general level as well as at an Azure level.  Finally, we have a [data and code](Data_and_Code.zip) set that will need to be uploaded to the Teams channel for the students to download and reference in the upcoming challenges. 
```

Comment:
You should be careful here... don't paste in redundant text here from the student challenge 0, it will main maintenance harder. Remember the Coaches can see the student challenge guides and worst case you can link to them from here.

Reply:
Intro part is different in prereqs.md file for both student and coach. 5 prerequisites are common for both.